src/jira/util/jwt.ts

```diff
@@ -184,6 +184,19 @@ export const verifySymmetricJwtTokenMiddleware = (secret: string, tokenType: Tok
 
 const ALLOWED_BASE_URLS = [BASE_URL]
 
+const isStagingTenant = (req: Request): boolean => {
+	let isStagingTenant = false;
+	const hostBaseUrl = _.get(req.body, "baseUrl");
+	if (hostBaseUrl) {
+		const host = new URL(hostBaseUrl).hostname;
+		const hostEnvironment = host.substring(host.indexOf(".") + 1);
```

Comment:
Couldn't you use regex for this? `return /\.jira-dev\.com$/.test(new URL(hostBaseUrl).host)`